docs/development/pigs/pig-013.rst

```diff
@@ -15,19 +15,25 @@ PIG 13 - Gammapy dependencies and distribution
 Abstract
 ========
 
-As preparation for the planned Gammapy v1.0 release fall 2019, we propose to
-clarify which Gammapy dependencies are required and which ones are optional. We
-also propose to increase the minimum required version for some dependencies.
-Finally, we propose some updates to the Gammapy supported distribution channels
-and installation instructions.
-
-This represents that plan for 2019 to be implemented before v1.0. Things are
-likely to evolve in the coming years, as Gammapy supports different projects,
-especially CTA.
+This is a proposal to make some updates concerning Gammapy dependencies and
+supported distribution channels, to be introduced in Gammapy v0.14, and then to
+be used in Gammapy v1.0 and likely without much changes moving forward.
+
+Given that we have a good way to ship recent versions of Gammapy and all
+dependencies to users on all platforms, we suggest to drop support for Python
+3.5 or later, and to increase the minimum supported version of our dependencies.
+This PIG also contains a table clarifying which Gammapy dependencies are
+required and which are optional, and what the minimum supported version for each
+dependency is (older versions often will work, but are not tested by us).
+
+Concerning distribution, we suggest to try to build and distribute Python binary
+wheels to get a fast ``pip install gammapy``, and if it works well and the extra
+maintenance effor is low, to support that distribution channel for Gammapy v1.0
+and moving forward. We suggest to remove support for distribution via Macports.
```

Comment:
> We suggest to remove support for distribution via Macports.

I would suggest to change the sentence to leave the option open for 3rd party packagers to package up gammapy, e.g. it'll stay in debian because they support it, not because gammapy devs support it --> "Gammapy developers do not actively support any other distribution channels."

Reply:
Yes, important point. I put a new sentence.